src/cnd.h

```diff
@@ -0,0 +1,19 @@
+#ifndef RLANG_CND_H
+#define RLANG_CND_H
+
+#include <stdbool.h>
+
+
+void r_inform(const char* fmt, ...);
+void r_warn(const char* fmt, ...);
+void r_abort(const char* fmt, ...);
+
+SEXP interp_str(const char* fmt, ...);
```

Comment:
What do you mean? Functions exported to R side are prefixed with `rlang_` but we do it in the init file.